search-term/readme.md

```diff
@@ -0,0 +1,16 @@
+#Term Generation
+
+When calling on getTermList, supply the Art and Architecture Thesaurus(AAT) ID
+
+Function takes the id and retrieves the hierarchy listing of terms, including all parent terms and two levels of children terms. After removing all extra formatting and html tags, function compiles the hierarchy list as a csv list.
+
+#Example Use
+
+
+Using the ID for marble (rock)
+
+`getTermList(300011443)`
```

Comment:
FYI, with the changes to the xml (test versions can be found here:  https://drive.google.com/drive/u/0/folders/1dmyElc0cuunBfDo7geaspMlNU4rtH-lq), we are now including the AAT URI directly in the xml.  Work still needs to be done to resolve the CONA urls.  Bottom line, you may have the URI already defined, and not need to create the URL yourself.